spec/spec_helper.rb

```diff
@@ -34,6 +36,14 @@
     Rails.cache.clear
   end
 
+  config.before(:each, type: :system) do
+    driven_by :rack_test
+  end
+
+  config.before(:each, type: :system, js: true) do
+    driven_by :apparition
+  end
+
   config.after(:each, type: :feature) do |example|
```

Comment:
This should be switched to `type: :system`. Also, we probably have other similar things happening for `:feature` test only in `solidus_dev_support`. Maybe it's better to change that gem as first step?

Reply:
I forgot that part. Yes, I need to update it.
But about switching to system specs in dev_support... I'm not convinced.
Some people could think to do this change at another time (or to skip it completely).

Reply:
Sorry, I didn't mean switching but just adding support for both system and feature test in `solidus_dev_support`. Using one or another will be a plugin choice

Reply:
Proposed a change here: https://github.com/solidusio/solidus_dev_support/pull/92